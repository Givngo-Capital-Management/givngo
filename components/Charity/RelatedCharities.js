import React from 'react';
import CharityPreview from './CharityPreview';

export default function RelatedCharities() {
  return (
    <div className="related-charities">
      <CharityPreview horizontal />
      <CharityPreview horizontal />
      <CharityPreview horizontal />
    </div>
  );
}
