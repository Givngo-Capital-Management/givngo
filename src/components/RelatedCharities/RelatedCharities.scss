.related-charities {
  & > * {
    margin-top: 4px;
  }
}