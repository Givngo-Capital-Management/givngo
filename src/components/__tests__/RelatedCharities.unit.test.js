import React from 'react';
import { shallow } from 'enzyme';
import RelatedCharities from '../Charity/RelatedCharities';

describe('RelatedCharities', () => {
  test('renders', () => {
    const wrapper = shallow(<RelatedCharities />);
    expect(wrapper).toMatchSnapshot();
  });
});
