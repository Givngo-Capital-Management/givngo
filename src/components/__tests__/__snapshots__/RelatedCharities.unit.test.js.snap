// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedCharities renders 1`] = `
<div
  className="related-charities"
>
  <CharityPreview
    horizontal={true}
  />
  <CharityPreview
    horizontal={true}
  />
  <CharityPreview
    horizontal={true}
  />
</div>
`;
